feconf.py

```diff
@@ -644,6 +644,8 @@ def get_empty_ratings():
 TOPIC_EDITOR_DATA_URL_PREFIX = '/topic_editor_handler/data'
 TOPIC_EDITOR_URL_PREFIX = '/topic_editor'
 TOPIC_MANAGER_RIGHTS_URL_PREFIX = '/rightshandler/assign_topic_manager'
+TOPIC_STATUS_URL_PREFIX = '/rightshandler/change_topic_status'
+TOPIC_RIGHTS_URL_PREFIX = '/rightshandler/get_topic_rights'
```

Comment:
Swap the above two lines with each other to preserve alphabetical order of constant names.

Reply:
Done.